tests.py

```diff
@@ -138,6 +139,21 @@ def test_set_password(self):
 
             frozen_time.move_to("2020-05-22 12:00:00")
             self.assertIsNone(snappass.get_password(key))
+    
+    def test_set_password_json(self):
+        with freeze_time("2020-05-08 12:00:00") as frozen_time:
+            password = 'my name is my passport. verify me.'
+            rv = self.app.post('/', data={'password': password, 'ttl': 'two weeks', 'json': True})
+
+            json_content = rv.data.decode("ascii")
```

Comment:
JSON should be treated as UTF-8, but that aside, there's a simple approach you can use here:

```suggestion
            json_content = rv.get_json()
```

... and then you don't need to use `json.loads()` below.